rewrite-java-8/src/main/java/org/openrewrite/java/ReloadableJava8JavadocVisitor.java

```diff
@@ -967,5 +968,21 @@ public J visitPrimitiveType(PrimitiveTypeTree node, Space fmt) {
             cursor += name.length();
             return new J.Identifier(randomId(), fmt, Markers.EMPTY, name, typeMapping.primitive(primitiveType.typetag), null);
         }
+
+        @Override
+        public J visitParameterizedType(ParameterizedTypeTree node, Space fmt) {
+            J.Identifier id = (J.Identifier) javaVisitor.scan(node.getType(), Space.EMPTY);
+            List<JRightPadded<Expression>> expressions = new ArrayList<>(node.getTypeArguments().size());
+            cursor += 1; // skip '<'
+            for (int i = 0; i < node.getTypeArguments().size(); i++) {
+                if (i > 0) {
+                    cursor++; // skip ','
```

Comment:
It's probably possible for there to be space before the `,`. In general we use `sourceBefore` for this purpose in parsing.